airbyte-webapp/src/components/connectorBuilder/types.ts

```diff
@@ -42,6 +52,11 @@ export const convertToManifest = (values: BuilderFormValues): ConnectorManifest
           name: stream.name,
           url_base: values.global?.urlBase,
           path: stream.urlPath,
+          request_options_provider: {
+            request_parameters: Object.fromEntries(stream.requestOptions.requestParameters),
+            request_headers: Object.fromEntries(stream.requestOptions.requestHeaders),
+            request_body_data: Object.fromEntries(stream.requestOptions.requestBody),
```

Comment:
I liked the idea with the toggle between json and url encoded, are you planning to do this in a separate PR?

Reply:
Yeah I am planning to do that separately. I want to check with the rest of the team on that though, and also ask them about if we want to have a generic JSON blob input for the request body since it seems like that is supported by the framework as well. So I will create a ticket and bring it to backlog grooming

Reply:
Created an issue here: https://github.com/airbytehq/airbyte/issues/20548